bundles/org.openhab.binding.xmltv/src/main/java/org/openhab/binding/xmltv/internal/XmlTVHandlerFactory.java

```diff
@@ -110,6 +120,8 @@ private synchronized void registerDeviceDiscoveryService(XmlTVHandler bridgeHand
 
     private synchronized void unregisterDeviceDiscoveryService(Thing thing) {
```

Comment:
Rather use ThingHanflerService to simplify all this.

Reply:
Done